### PR TITLE
ztp: Update PolicyGen to create static inform policy

### PR DESCRIPTION
**Pull request by Missxiaoguo**
Move the CRs need to be validated into a source YAML file
and update example common PGT to create a inform policy
to validate configuration CRs to help indicate when ztp
of a cluster is done

Jira: https://issues.redhat.com/browse/CNF-2960
Signed-off-by: Angie Wang <angwang@redhat.com>

### CODE REVIEW

File: ztp/policygenerator/policyGen/policyBuilder.go [hunk 1]
```diff
@@ -85,6 +87,29 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 				policies[output] = acmPolicy
 			}
 		}
+
+		for key := range policyGenTemp.Spec.BindingRules {
+			// Bind static ACM validator inform policy to 'common' PGT
+			match, err := regexp.MatchString("[c|C]ommon|COMMON", key)
```
Comment (serngawy) [inline ztp/policygenerator/policyGen/policyBuilder.go:93]: are we grantee that "common" key word will not get change ?

**Review by serngawy — CHANGES_REQUESTED** [review]
its better to put this on hold till. complainceType & remedationAction PRs get merged, you will need this work.

**Comment by Missxiaoguo** [comment]
Commit is updated. As we discussed, leverage PGT to create validator inform policy.
/cc @serngawy @imiller0

File: ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml [hunk 2]
```diff
@@ -7,6 +7,11 @@ spec:
   bindingRules:
     common: "true"
   sourceFiles:
+    # Create inform policy to validate configuration CRs that will be applied to all clusters
+    - fileName: validatorCRs/informDuValidator.yaml
+      complianceType: musthave
+      remediationAction: inform
+      policyName: "du-validator-policy"
```
Comment (imiller0) [inline ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml:14]: I don't think we should be copying all the published content into the test data directory. This adds confusion as to where the active code really is. If the added YAML presents a new type of input then creating generic test data here makes sense. In this case it looks like policy with remediationType of "inform" is new and it makes sense to add in tests to cover that.

Comment (Missxiaoguo) [reply ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/common-ranGen.yaml]: Updated.

**Review by serngawy — APPROVED** [review]

**Pull request merged**